src/common/dependency-manager/local-manager.ts

```diff
@@ -40,7 +40,7 @@ export default class LocalDependencyManager extends DependencyManager {
     const dependency_resolvers = [];
     for (const [ref, env_svc_cfg] of Object.entries(dependency_manager.environment.getServices())) {
 
-      if (env_svc_cfg.host && env_svc_cfg.port) {
+      if ((env_svc_cfg.host && env_svc_cfg.port) || env_svc_cfg.interfaces) {
```

Comment:
This is exactly what we talked about, but it does look wrong. 

Maybe all nodes should have interfaces instead of a list of ports? Hides behind a bush.

Reply:
Ports are so widespread throughout our stack currently that it might be a whole new ticket (tickets?) in itself to make that conversion. I really like the idea though, as maybe ports were too simple and inflexible. It's also pretty weird that for this ticket I (read: David) ran into a bug where the port of the service config was conflicting with the interfaces. I would love for that never to be able to happen again

Reply:
Have interfaces be a getter that returns a default interface if none are defined - mirror service node.

Reply:
Ignore the other comment about making ports interfaces